src/website/views/RayeContainer.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import CustomAccordion from './components/DropDown/DropDown';
+import { styled } from '@mui/material/styles';
+import {
+  downloadGPSData,
+  downloadAISShipsData,
+  downloadGlobalPathData,
+  downloadLocalPathData,
+  downloadBatteriesData,
+  downloadWindSensorsData,
+  downloadGenericSensorsData
+} from './components/DownloadData/DownloadData';
+
+const textAndTitleColor = '#092e4e';
+
+const Container = styled('div')(({ theme }) => ({
```

Comment:
Same request, could we move everything in here dealing with styles to a `.css` file?